pkg/asset/cluster/tfvars.go

```diff
@@ -475,26 +476,28 @@ func (t *TerraformVariables) Load(f asset.FileFetcher) (found bool, err error) {
 // injectInstallInfo adds information about the installer and its invoker as a
 // ConfigMap to the provided bootstrap Ignition config.
 func injectInstallInfo(bootstrap []byte) (string, error) {
+	openshiftInstallPath := filepath.Join("opt", "openshift", openshiftinstall.ConfigPath)
```

Comment:
Relative path is being passed here, this breaks ignition

Reply:
Good catch. Thanks.